lib/cucumber/cli/argument_parser.js

```diff
@@ -106,9 +107,19 @@ function ArgumentParser(argv) {
       return modules;
     },
 
-    getFormat: function getFormat() {
-      var format = self.getOptionOrDefault(ArgumentParser.FORMAT_OPTION_NAME, ArgumentParser.DEFAULT_FORMAT_VALUE);
-      return format;
+    getFormats: function getFormats() {
+      var formats = self.getOptionOrDefault(ArgumentParser.FORMAT_OPTION_NAME, [ArgumentParser.DEFAULT_FORMAT_VALUE]);
+      var outputMapping = {};
+      formats.forEach(function (format) {
+        var parts = format.split(':');
+        var type = parts[0];
+        var outputTo = parts[1] || '';
+        outputMapping[outputTo] = type;
+      });
+      return _.map(outputMapping, function (type, outputTo) {
+        var stream = outputTo ? fs.createWriteStream(outputTo) : process.stdout;
```

Comment:
@charlierudolph could you add a test for that case as well? Cucumber's return code should also be non-zero.